plugins/qualified.js

```diff
@@ -0,0 +1,29 @@
+module.exports = async function fullStory({ context, options }) {
```

Comment:
assuming this block was inspired by the fullstory implementation, but the function should get it's own name 😄